src/test/groovy/CountryFactoryTest.groovy

```diff
@@ -0,0 +1,45 @@
+import com.covid.api.dto.CountryData
+import com.covid.api.dto.CovidSummaryResponse
+import com.covid.api.dto.GlobalData
+import com.covid.api.factory.CountryFactory
+import spock.lang.Specification
+
+import java.time.LocalDateTime
+
+class CountryFactoryTest extends Specification {
+
+    def factory = new CountryFactory()
+
+    def "assembleCountries() should return a list of countries with correct data"() {

```

Comment:
`assembleCountries()` is not needed, in the when block i should have all this information. `should` is a good start word, i do it in the same way ;)